File: Easy/C/Grade the Students.c
#include <inttypes.h>
#include <stdio.h>

int main() {
    uint8_t T;
    scanf("%" SCNu8, &T);
    for(uint8_t i = 0; i < T; ++i) {
        uint8_t Mathematics, Algorithms;
        scanf("%" SCNu8 " %" SCNu8, &Mathematics, &Algorithms);
        printf("%s\n", Mathematics > 70 && Algorithms > 50 ? "Pass" : "Fail");
    }
}

File: Easy/Rust/Grade the Students.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let mut lines = stdin.lock().lines();
    let T: u8 = lines.next().unwrap().unwrap().trim().parse().unwrap();
    for _ in 0..T {
        let line = lines.next().unwrap().unwrap();
        let mut Marks = line.split_whitespace();
        let Mathematics: u8 = Marks.next().unwrap().parse().unwrap();
        let Algorithms: u8 = Marks.next().unwrap().parse().unwrap();
        match Mathematics {
            0..=70 => println!("Fail"),
            _ => match Algorithms {
                0..=50 => println!("Fail"),
                _ => println!("Pass"),
            },
        }
    }
}
